File: rohitMigrationTesting/pipelines/pipeline1/code/src/main/scala/abhishek0005prophecy/io_team/rohitmigrationtesting/pipeline/pipeline1/graph/DS1.scala
package abhishek0005prophecy.io_team.rohitmigrationtesting.pipeline.pipeline1.graph

import io.prophecy.libs._
import abhishek0005prophecy.io_team.rohitmigrationtesting.pipeline.pipeline1.config.Context
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object DS1 {

  def apply(context: Context): DataFrame =
    context.spark.read
      .format("csv")
      .option("header",                                   true)
      .option("sep",                                      ",")
      .schema(StructType(Array(StructField("customer_id", StringType, true))))
      .load("dede")

}

File: MigrationPy/pipelines/p2/code/p1/graph/ds1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from prophecy.libs import typed_lit
from prophecy.transpiler import call_spark_fcn
from prophecy.transpiler.fixed_file_schema import *
from p1.config.ConfigStore import *
from p1.udfs.UDFs import *

def ds1(spark: SparkSession) -> DataFrame:
    return spark.read\
        .schema(StructType([StructField("id", StringType(), True), StructField("first_name", StringType(), True)]))\
        .option("header", True)\
        .option("sep", ",")\
        .csv("dbfs:/Prophecy/rohitjain+27@simpledatalabs.com/CustomersDatasetInput.csv")
